scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram.py

```diff
@@ -316,7 +325,15 @@ def default_autocast():
 
         with autocast():
             with torch.no_grad():
-                all_logits = asr_model.transcribe(audio_file_paths, batch_size=cfg.acoustic_batch_size, logprobs=True)
+                if isinstance(asr_model, EncDecHybridRNNTCTCModel):
```

Comment:
The EncDecHybridRNNTCTCModel and regular ctc models should return the same number of outputs to be consistent.